src-tauri/src/main.rs

```diff
@@ -85,66 +82,70 @@ async fn main() {
         .and_then(|s| log::LevelFilter::from_str(s).ok())
         .unwrap_or(log::LevelFilter::Info);
 
-    wooting_macro_backend::MacroBackend::generate_directories();
+    MacroBackend::generate_directories().expect("unable to generate config directories");
```

Comment:
Tried replacing all unwraps with expects where possible and also added some due to error handling propagation

Reply:
expect is nice, but shouldn't you be able to make the main function have a Result return type so that you can just do `?`. Then you should be able to add the context with anyhow

Reply:
Could not do it everywhere as closures can't work with anyhow, but the top level expects have been replaced with anyhow context in a fix commit coming later

Reply:
That's better